src/config.sh

```diff
@@ -223,6 +228,11 @@ function parse_config_options()
         options_values['SCOPE']='local'
         shift
         ;;
+      --show | -s)
+        shift 2
+        show_configurations "$@"
```

Comment:
I think it would be better if you use something like `options_values['SHOW_CONFIG']+="$2"`. I guess you'll need to check if `$2` is empty; if so, just add 1 to the variable value.